packages/ui-mobile-base/android/widgets/src/main/java/org/nativescript/widgets/Utils.java

```diff
@@ -36,8 +36,14 @@
 
 public class Utils {
 	public static Drawable getDrawable(String uri, Context context){
-		String resPath = uri.substring("res://".length());
-		int resId = context.getResources().getIdentifier(resPath, "drawable", context.getPackageName());
+		int resId = 0;
+		int resPrefixLength = "res://".length();
+		
+		if (uri.length() >= resPrefixLength) {
```

Comment:
I think we may want the check to be `>` since `>=` would make `restPath` an empty string in case the `uri` is `res://`

```suggestion
		if (uri.length() > resPrefixLength) {
```

Reply:
You're right, it will be more performant like this. No need to try to load an empty path!